js/controller.js

```diff
@@ -36,8 +36,11 @@ class TrickleController extends Backbone.Controller {
 
   async onDataReady() {
     await data.whenReady();
-    addButtonComponents();
-    applyLocks();
+    Adapt.wait.for((done) => {
```

Comment:
Please remove extra brackets
```suggestion
    Adapt.wait.for(done => {
```